amostra/server/engine.py

```diff
@@ -338,13 +338,13 @@ def post(self):
             uids.append(data['uid'])
             res = database.container.insert(data)
             if res:
-                database.request.create_index([('uid', pymongo.DESCENDING)],
-                                                 unique=True, background=True)
-                database.request.create_index([('time', pymongo.DESCENDING)],
-                                            unique=False, background=True)
+                database.container.create_index([('uid', pymongo.DESCENDING)],
```

Comment:
Don't let create_index() mislead you. They deprecated ensure_index. So this is essentially both create and ensure.